File: projetPfa/chatEtInvitation/API/DTOs/AmisChatReturnedDTO.cs
namespace chatEtInvitation.API.DTOs
{
    public class AmisChatReturnedDTO
    {
        public int Id { get; set; }
        public string AmiName { get; set; } 
        public int? idMember { get; set; }
        public string LastMessage { get; set; }
        public DateTime Date { get; set; }
        public int? UnreadCount { get; set; }
        public string Statut { get; set; }
        public string Avatar { get; set; }
        
    }
}

File: projetPfa/angular/src/app/core/models/chat/amis-chat-returned.dto.ts
export interface AmisChatReturnedDTO {
    id: number;
    amiName: string;
    lastMessage: string;
    date: Date;
    unreadCount?: number;
    statut?: string;
    avatar: string;
    idMember:string;
    type: 'amis'; 
  }

 export interface PaginatedResponse<T> {
    page: number;
    pageSize: number;
    totalCount: number;
    items: T[];
  }